dcpy/lifecycle/scripts/validate_ingest.py

```diff
@@ -53,6 +56,12 @@ def convert_template(dataset: str):
     converter = Converter(
         id=library_template["dataset"]["name"],
         acl=library_template["dataset"]["acl"],
+        attributes=DatasetAttributes(
+            _exclude_falsey_values=False,
+            name=library_template["dataset"]["name"],
```

Comment:
Should I just make this an empty string? Something that a little more clearly needs to be changed

Reply:
IIRC I used to just do something like "< CHANGE ME >" in metadata

Reply:
Yeah I'll just do that